docs/operator-guide/troubleshooting/troubleshoot-container-registries.md

```diff
@@ -0,0 +1,25 @@
+# Container Registry Troubleshooting
+
+## Problem
+
+Reset container registry is not feasible due to the `Change registry` button not working.
+
+## Cause
+
+The External Secrets Operator (ESO) doesn't allow you to reconfigure registry.
+
+## Solution
+
+Remove the `kaniko-docker-config` and `regcred` resources from both ESO and secrets.
```

Comment:
Remove the `kaniko-docker-config` and `regcred` resources from both the External Secrets custom resources and the Kubernetes secrets.

Reply:
Remove the `kaniko-docker-config` and `regcred` resources from both the ExternalSecret custom resources and the Kubernetes secrets.